client/src/components/SuiteOptionsPage/SuiteOptionsPage.tsx

```diff
@@ -89,103 +78,74 @@ const SuiteOptionsPage: FC<SuiteOptionsPageProps> = ({ testSuite }) => {
-          {suiteOption.title}
-          {suiteOption.description && (
-            <Tooltip title={suiteOption.description}>
-              <HelpOutlineOutlinedIcon fontSize="small" color="secondary" sx={{ px: 0.5 }} />
-            </Tooltip>
-          )}
-        </FormLabel>
-
-        <RadioGroup
-          aria-label={`suite-option-group-${suiteOption.id}`}
-          defaultValue={
-            suiteOption.list_options &&
-            suiteOption.list_options.length &&
-            suiteOption.list_options[0].value
-          }
-          name={`suite-option-group-${suiteOption.id}`}
-        >
-          {suiteOption?.list_options?.map((choice, k) => (
-            <FormControlLabel
-              value={choice.value}
-              control={<Radio size="small" />}
-              label={choice.label}
-              key={`radio-button-${k}`}
-              onClick={() => {
-                changeSuiteOption(suiteOption.id, choice.value);
-              }}
-            />
-          ))}
-        </RadioGroup>
-      </FormControl>
+      <Typography
+        variant="h3"
+        component="h1"
+        align="center"
+        sx={{
+          color: lightTheme.palette.common.grayDarkest,
+          fontSize: windowIsSmall ? '2rem' : 'auto',
+          fontWeight: 'bolder',
+          letterSpacing: 2,
+        }}
+      >
+        {testSuite?.title.toUpperCase()}
+      </Typography>
     );
   };
 
   return (
-    <Box
-      display="flex"
-      alignItems="center"
-      justifyContent="center"
-      flexDirection="column"
-      minHeight="600px"
-      height="100%"
-      maxHeight="100vh"
+    <Container
+      maxWidth={false}
       role="main"
+      className={classes.main}
+      sx={
+        windowIsSmall
+          ? {}
+          : {
+              flexDirection: 'column',
+              minHeight: '400px',
+              maxHeight: '100vh',
+              py: 10,
+            }
+      }
     >
       <Box
         display="flex"
-        flexWrap="wrap"
+        flexDirection="column"
+        justifyContent="center"
         alignItems="center"
-        justifyContent="space-evenly"
-        width="100%"
-        sx={windowIsSmall ? { overflow: 'auto' } : { mt: 4 }}
+        maxWidth={windowIsSmall ? '100%' : '50%'}
+        maxHeight={windowIsSmall ? 'none' : '100%'}
+        minHeight={windowIsSmall ? 'none' : '100%'}
+        overflow="auto"
+        px={windowIsSmall ? 0 : 8}
+        my={3}
       >
+        {/* Title */}
         <Box
+          display="flex"
+          alignItems="center"
+          sx={windowIsSmall ? { mx: 2 } : { m: 4 }}
           maxWidth={descriptionWidth}
           maxHeight={windowIsSmall ? 'none' : '100%'}
+        >
+          {renderTitle()}
+        </Box>
+        {/* Description */}
+        <Box
+          display="flex"
+          maxWidth={descriptionWidth}
+          justifyContent="center"
           overflow="auto"
```

Comment:
This may be too minor to require any changes, but the responsiveness seems slightly off with the description box. It will switch to a scroll view if the window height is shortened, but then it won't switch back out of the scroll view if you make window tall enough again, and the box keeps the max height of the scroll view.

Here's what I mean:
https://github.com/inferno-framework/inferno-core/assets/8235974/6469ad48-2c07-44e2-9597-33c4e313fbd3

Maybe that's just the behavior you get with mui? Not sure.

Reply:
Good catch; this seems to be a Firefox issue (not a problem in Chrome) so I'll have to do some digging. The codecov is something we can ignore for now as long as the actual functionality is there. The work I'm currently doing is intended to bring the frontend coverage up to speed. Thanks!

Reply:
Ok I think we can leave this bug as I can't reproduce it in the follow-up that removes the Suite Options Page, so it will eventually go away.

Reply:
Works for me. Thanks for looking into it.